flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/agg/HashAggITCase.scala

```diff
@@ -80,8 +80,20 @@ class HashAggITCase(adaptiveLocalHashAggEnable: Boolean)
     )
   }
 
-  @Test
-  def testAdaptiveLocalHashAggWithLowAggregationDegree(): Unit = {
+  @ParameterizedTest
+  @MethodSource(Array("parameters"))
+  def testAdaptiveLocalHashAggWithLowAggregationDegree(
+      adaptiveLocalHashAggEnable: Boolean): Unit = {
+    if (adaptiveLocalHashAggEnable) {
+      tEnv.getConfig
+        .set(OptimizerConfigOptions.TABLE_OPTIMIZER_AGG_PHASE_STRATEGY, "TWO_PHASE")
+      tEnv.getConfig.set(
+        HashAggCodeGenerator.TABLE_EXEC_LOCAL_HASH_AGG_ADAPTIVE_ENABLED,
+        Boolean.box(true))
+      tEnv.getConfig.set(
+        HashAggCodeGenerator.TABLE_EXEC_LOCAL_HASH_AGG_ADAPTIVE_SAMPLING_THRESHOLD,
+        Long.box(5L))
+    }
```

Comment:
Because there are too many duplicated lines for the set envs, we can extract it with a new method like this. And all the methods can call the new method.

  private def setTableEnv(adaptiveLocalHashAggEnable: Boolean) = {
    if (adaptiveLocalHashAggEnable) {
      tEnv.getConfig
        .set(OptimizerConfigOptions.TABLE_OPTIMIZER_AGG_PHASE_STRATEGY, "TWO_PHASE")
      tEnv.getConfig.set(
        HashAggCodeGenerator.TABLE_EXEC_LOCAL_HASH_AGG_ADAPTIVE_ENABLED,
        Boolean.box(true))
      tEnv.getConfig.set(
        HashAggCodeGenerator.TABLE_EXEC_LOCAL_HASH_AGG_ADAPTIVE_SAMPLING_THRESHOLD,
        Long.box(5L))
    }
  }

Reply:
Done.